pkg/odo/cli/url/create.go

```diff
@@ -54,7 +54,7 @@ var (
 	%[1]s --host apps-crc.testing --ingress
 
 	# Create a secured URL for the current component with a specific host (using CRC as an example)
```

Comment:
if you can replace in comment comment also.

Reply:
updated, thanks for reminder